cypress/support/commands.ts

```diff
@@ -181,6 +181,7 @@ Cypress.Commands.add(
     differentStartTime = "",
   }) => {
     cy.visit('contest/new/');
+    cy.get('.introjs-skipbutton').click();
```

Comment:
Esto si se va a ocupar por que es cuando se va a crear un concurso nuevo. 

Pero existe una prueba (`Should create two contest and merge their scoreboard`) donde se crean dos concursos. Quiere decir que el modal sólo se mostrará cuando se cree el primero de ellos. 

Entonces aquí hay que agregar una condición  mas o menos así:
```suggestion
    if (shouldShowIntro) {
      cy.get('.introjs-skipbutton').click();
    }
```
Para que esto funcione hay que recibir en la función un segundo argumento:
```ts
shouldShowIntro: boolean = true
```
configurado por defecto en `true` para no tener que mover el resto de pruebas y sólo modificar la que nos interesa.

Hay otra función en el archivo `contestPage.ts` también llamada `createContest` que debería recibir también el parámetro que agregamos aquí.

Y por último, hay que agregar este nuevo parámetro en el archivo de tipos `cypress.d.ts` en la declaración de la función `createContest`

Reply:
No agregaste el nuevo argumento en las funciones que te indiqué :(